aws-android-sdk-appsync/src/main/java/com/amazonaws/mobileconnectors/appsync/ClearCacheOptions.java

```diff
@@ -0,0 +1,103 @@
+/**
+ * Copyright 2018-2019 Amazon.com,
+ * Inc. or its affiliates. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.amazonaws.mobileconnectors.appsync;
+
+/**
+ * Options object that indicates what client databases (caches)
+ * to clear when passed in to AWSAppSyncClient#clearCache(ClearCacheOptions).
+ */
+public class ClearCacheOptions {
+    private boolean queries;
+    private boolean mutations;
+    private boolean subscriptions;
+
+    /**
+     * Constructor invoked by the builder.
+     *
+     * @param queries true indicates clear the query responses in query cache
+     * @param mutations true indicates clear the offline persistent mutations
+     * @param subscriptions true indicates clear the subscriptions metadata stored by Delta Sync
+     */
+    private ClearCacheOptions(boolean queries, boolean mutations, boolean subscriptions) {
+        this.queries = queries;
+        this.mutations = mutations;
+        this.subscriptions = subscriptions;
+    }
+
+    /**
+     * Builder class used for constructing ClearCacheOptions object.
+     */
+    public static class Builder {
```

Comment:
A builder for this is unnecessary.  Just use the constructor.

Reply:
We decided to use the builder during the API review because in the future we may add more caches and we don't want to overload the constructor(s).

Reply:
sounds good.